src/Block/PageListBlockService.php

```diff
@@ -101,11 +104,4 @@ public function configureSettings(OptionsResolver $resolver): void
             'template' => '@SonataPage/Block/block_pagelist.html.twig',
         ]);
     }
-
-    public function getBlockMetadata($code = null)
```

Comment:
You need to rename it `getMetadata`, no ?

Reply:
And this class need now to implement EditableBlockInterface

With
```
    public function configureCreateForm(FormMapper $form, BlockInterface $block): void
    {
        $this->configureEditForm($form, $block);
    }
```
and 
```
    public function validate(ErrorElement $errorElement, BlockInterface $block): void
    {
    }
```

Reply:
so we have to remove buildEditForm ?

Reply:
No, you have to add `implements EditableBlockService`, keep buildEditForm and add both configureCreateForm and validate